app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons_list = [Person(person["name"], person["age"]) for person in people]
+    for personal_info in people:
+        if personal_info.get("wife"):
+            Person.people[personal_info["name"]].wife = \
+                Person.people[personal_info["wife"]]
```

Comment:
get rid of those line breaks. You have almost always avoid using them. You can use additional variables to do so